archetypes/blog/index.md

```diff
@@ -0,0 +1,13 @@
+---
+date: '{{ .Date }}'
+lastmod: :git
+author: someone@example.com
+title: '{{ replace .File.ContentBaseName `-` ` ` | title }}'
+description: A short description does here.
+tags: []
+hero: false
```

Comment:
Do we want hero set in a file? Seems to make more sense it can be set in a single place to avoid multiple articles having it set to true or even the pain of finding the file to remove it?